scripts/helper/run_all.sh

```diff
@@ -156,13 +142,19 @@ declare -a directories
 # read input arguments
 read_arguments "$@"
 
+# temporarily overwrite the log directory
+export ROSWSS_LOG_DIR="${LOG_DIR}"
```

Comment:
I'm a little bit concerned about possible race conditions here. Just handover the log path to the `run_scripts ` and explicitly set it for the screen call, e.g. `env ROSWSS_LOG_DIR=${MY_PARAM} roswss screen ...`.

See: https://unix.stackexchange.com/questions/495161/import-environment-variables-in-a-bash-script